website_sale_tax_toggle/controllers/main.py

```diff
@@ -8,10 +9,38 @@
 
 
 class WebsiteSaleTaxToggle(WebsiteSale):
+    @http.route()
+    def shop(
+        self,
+        page=0,
+        category=None,
+        search="",
+        min_price=0.0,
+        max_price=0.0,
+        ppg=False,
+        **post
+    ):
+        res = super().shop(
+            page=page,
+            category=category,
+            search=search,
+            min_price=min_price,
+            max_price=max_price,
+            ppg=ppg,
+            **post
+        )
+        default_tax_toggle = request.website.default_tax_toggle
+        if request.session.get("tax_toggle_taxed") is None:
```

Comment:
Don't read the value if not needed:

```suggestion
        if request.session.get("tax_toggle_taxed") is None:
            default_tax_toggle = request.website.default_tax_toggle
```

Reply:
All changes done